public/js/graph/react_graph.js

```diff
@@ -918,7 +1009,9 @@ var Edge = React.createClass({
                      this.props.svg.refs.bools.refs[this.props.source];
         var target = this.props.svg.refs.nodes.refs[this.props.target] ||
                      this.props.svg.refs.bools.refs[this.props.target];
-        if (!source.isSelected()) {
+        if (target.state.status === 'missing') {
```

Comment:
This isn't bad, but will fail if this edge is takeable, but the target has another prerequisite which is missing.

You need to check both the state of the target as well as whether the source is not selected.